application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksCommand.java

```diff
@@ -0,0 +1,145 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.unions.MessageChannelUnion;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+
+import java.util.Objects;
+
+public final class BookmarksCommand extends SlashCommandAdapter {
```

Comment:
missing doc on probably the most important class in ur PR

Reply:
I would not consider this the most important class as it is just the frontend but you are correct

Reply:
thats certainly true. its just that its the core entry point of the feature. so id expect this to be the place where i get to see an overview of all supported commands/features.

Reply:
Changed